RadixWallet/Clients/AppsFlyerClient/AppsFlyerClient+Live.swift

```diff
@@ -37,16 +37,13 @@ extension AppsFlyerClient: DependencyKey {
 	}
 
 	private class Delegate: NSObject, DeepLinkDelegate, @unchecked Sendable {
+		@Dependency(\.appEventsClient) var appEventsClient
+
 		func didResolveDeepLink(_ result: DeepLinkResult) {
 			if let deepLink = result.deepLink {
 				loggerGlobal.info("did resolve deep link. Is deferred: \(deepLink.isDeferred). Click events: \(deepLink.clickEvent)")
-				if deepLink.isDeferred {
-					let message = if let deepLinkValue = deepLink.clickEvent["deep_link_value"] as? String {
-						"Resolved deferred DL with value \(deepLinkValue)"
-					} else {
-						"Resolved deferred DL without value"
-					}
-					AppsFlyerLib.shared().logEvent(message, withValues: deepLink.clickEvent)
+				if deepLink.isDeferred, let value = deepLink.clickEvent["deep_link_value"] as? String {
+					appEventsClient.handleEvent(.deepLinkReceived(value))
```

Comment:
mm I am not sure I prefer it this way, since `DeepLinkHandlerClient` is about non deferred deep links. The logic for deferred vs non deferred is totally different, so not sure if it wouldn't add more confusion to put it there just for the sake of being a deep link.